package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "3.10.2",
+  "version": "4.0.0",
```

Comment:
This feature is definitely a large, non-compatible change, so I think a major release change is appropriate.

Reply:
I agree, i think it's large enough to warrant version: `10.0.0` 😆